app/src/main/java/net/squanchy/schedule/SchedulePageView.kt

```diff
@@ -120,25 +124,51 @@ class SchedulePageView @JvmOverloads constructor(
     }
 
     fun updateWith(schedule: Schedule, onEventClicked: (Event) -> Unit) {
-        viewPagerAdapter.updateWith(schedule.pages, onEventClicked)
+        val initialEventForPage = schedule.pages.map(::findNextEventForPage).toTypedArray()
+        viewPagerAdapter.updateWith(schedule.pages, initialEventForPage, onEventClicked)
+
+        val todayPageIndex = findTodayIndexOrDefault(schedule.pages)
+        viewpager.setCurrentItem(todayPageIndex, false)
+
+        tabstrip.addOnTabSelectedListener(ScrollingOnTabSelectedListener(schedule, viewPagerAdapter))
         progressbar.visibility = View.GONE
     }
 
+    private fun findTodayIndexOrDefault(pages: List<SchedulePage>): Int {
+        val now = currentTime.currentLocalDateTime()
```

Comment:
True. Do we have a timezone for the whole conference? Otherwise, we can assume that the timezone of any event will be the timezone for the conference, and we can use that to calculate the days accurately.

Reply:
Yes it's retrieved from Firebase. You can see how in `ScheduleService`